tests/test_web_sendfile_functional.py

```diff
@@ -165,7 +165,9 @@ def handler(request):
     resp.close()
 
     resp = yield from client.get('/', headers={'If-Modified-Since': lastmod})
+    body = yield from resp.read()
     assert 304 == resp.status
+    assert b'' == body
```

Comment:
Thx!

This habit of so many dev to always use `.get()` 😕 This is so much more readable (and slightly more correct):

```python
assert "Content-Length" not in resp.headers
```

(Don't bother though, not important)